### DISCUSSION TITLE
Specify Python interpreter prior to installing Nevergrad

### DISCUSSION THREAD

**Discussion by LeondraJames** [facebookexperimental/Robyn#471] [General]
Just wanted to leave a recommendation to anyone who has an issue getting past the installation of Nevergrad, particularly if you're using RStudio. I recommend first going into **Tools --> Global Options --> Python**, then selecting your local interpreter. Finally, hit "**Apply**".  This will save you a headache prior to installing Nevergrad. Otherwise the console will ask if you'd like to use Miniconda (as it was unable to identify a local Python interpreter). It's much easier to use the Python interpreter you already have on your machine (if you already have it). It may be helpful for Meta developers to leave instruction if users do not already have Python installed.

I should note, I used the PIP method:
```
virtualenv_create("r-reticulate")
py_install("nevergrad", pip = TRUE)
use_virtualenv("r-reticulate", required = TRUE)
```
This issue may not be a problem with the Conda approach to installing Nevergrad.

**Comment by laresbernardo** [comment]
Thanks for sharing @LeondraJames
